userspace/libsinsp/sinsp.cpp

```diff
@@ -532,6 +532,11 @@ void sinsp::open_live_common(uint32_t timeout_ms, scap_mode_t mode)
 		oargs.gvisor_socket = m_gvisor_socket.c_str();
 		oargs.gvisor_root_path = m_gvisor_root_path.c_str();
 		oargs.gvisor_trace_session_path = m_gvisor_trace_session_path.c_str();
+	} else {
+		oargs.gvisor = false;
```

Comment:
Nit: Perhaps we could do `oargs.gvisor = m_gvisor` right underneath `oargs.udig = m_udig`? I'm fine either way.

Reply:
You are right, that would be a lot neater, I'll make the change real quick!